src/routes/users.js

```diff
@@ -29,7 +29,7 @@ router.post(
       return res.status(400).json({ errors: errors.array() });
     }
 
-    let { name, email, password } = req.body;
+    let { name, email, password, userImage } = req.body;
```

Comment:
When user register,there are no user Image field for them to fill. Users only update their user image when they decide to change profile pic after register and which is can be done inside the user component.  In this case there are no change in the register route, but we do need a patch/put route just for  changing image based on userId.